index.js

```diff
@@ -79,8 +76,7 @@ const DragSimulator = {
                 this.target.trigger('pointerup', {
                   which: 1,
                   button: 0,
-                  clientX,
-                  clientY,
+                  clientPosition,
```

Comment:
I think the `clientPosition` object should also be spreaded here:
```suggestion
                  ...clientPosition,
```

Reply:
You are right. I updated the PR. Thanks for taking care of this project.